src/material/_index.scss

```diff
@@ -14,6 +14,25 @@
 @forward './core/typography/typography-utils' show typography-level,
   font-size, line-height, font-weight, letter-spacing, font-family, font-shorthand;
 
+// Private/Internal
+@forward './core/density/private/all-density' as private-* show private-all-component-densities;
```

Comment:
Instead of exposing these through the main entry point, should we have a separate entry point instead?

Reply:
Yes, I could do that, but it has two consequences:

* Need for maintaining/wiring up another file in the NPM package (e.g. in the `exports`)
* Increase in inconsistency for cross-package imports (where currently there is just a single way)

Also people might still import from it. I believe there is not much benefit, or do you have other thoughts?